src/main/kotlin/org/wycliffeassociates/otter/common/persistence/repositories/IResourceRepository.kt

```diff
@@ -9,6 +9,7 @@ import org.wycliffeassociates.otter.common.data.workbook.ResourceInfo
 interface IResourceRepository : IRepository<Content> {
     fun getResources(collection: Collection, resourceInfo: ResourceInfo): Observable<Content>
```

Comment:
Previously, this function returned all resources about the collection (chapter) and all resources about the contents (chunks) within the chapter. I changed the implementation of this function to only supply the resources about the collection (chapter) and not its children. (See new function below.)